### PR TITLE
Our hero's courage waxes

### PR DESCRIPTION
**Pull request by ranjeevmahtani**
[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/1592)

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

File: english/break-through-door/break-through-door.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+You're confident that you can do better than any of those pitiful fools
```
Comment (casey-collab) [inline english/break-through-door/break-through-door.md]: It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.

**Comment by casey-collab** [comment]
It looks like the files `english/charge-a-marshmellow/charge-a-marshmellow.md` and `english/flee-marshmellows/flee-marshmellows.md` are empty.  Please make sure you don't add empty files to the repository.  You could add one sentence to each file, or delete the options that lead to them.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Comment by ranjeevmahtani** [comment]
Hi Caroline,

Thanks for the feedback on this. I've been traveling and haven't had a
chance to address the needed changes. I'll take care of them and reopen the
pull request then.

Thanks again!
Ranjeev

**Pull request closed without merging**